src/_utils/item/index.unit.tsx

```diff
@@ -66,6 +66,11 @@ describe('Item', () => {
     expect(wrapper.type()).toEqual('a')
     expect(wrapper.prop('href')).toEqual('#')
   })
+  it('Should return a <a /> tag without extra attribrutes when href is a string', () => {
+    const wrapper = shallow(<Item href="#" tag={<button type="button" />} />)
```

Comment:
The thing is, there is nothing that prevents this behaviour when using the component, outside of writing somewhere: "don't use them both at the same time". The whole `href` vs `tag` is a bit blurry, and to be honest I'd rather change those two props (and perhaps merge them into one), but I think this would need another refactoring ticket as this impacts a lot of different items as well.